<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wine_setup</class>
 <widget class="QDialog" name="wine_setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="wineLocationGroup">
     <property name="title">
      <string>Location of wine</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,10000">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="pathDefault">
        <property name="text">
         <string>System default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="pathCustom">
        <property name="text">
         <string>Custom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wineRootPath"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="wineCustomEnvGroup">
     <property name="title">
      <string>Specify a custom environment (PlayOnLinux/Mac)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1000,0" columnminimumwidth="0,300,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="winePrefix">
        <property name="text">
         <string>Wine prefix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="winePrefixPath"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="winePrefixBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="wineExec">
        <property name="text">
         <string>Wine executable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wineExecPath"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="wineExecBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="win64Exec">
        <property name="text">
         <string>Win64 executable</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="wine64ExecPath"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="wine64ExecBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="wineDll">
        <property name="text">
         <string>Wine DLL path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="wineDllPath"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="wineDllBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="importFromPlay">
     <property name="title">
      <string notr="true">---------------------------</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1000,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="playOnLinuxDrive"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="doImportFromPoL">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>358</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>wine_setup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>wine_setup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>wineCustomEnvGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>importFromPlay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
